### ISSUE TITLE
Bug Report: Missing Dark Mode Toggle

### ISSUE THREAD

**Issue by GARY121github** [layer5io/layer5#6309]
### Description
The toggle to switch between dark mode and light mode is missing from the UI. Users are unable to switch themes, which affects accessibility and user experience manually.

### Steps to Reproduce
- Visit the Layer5 Community Handbook.
- Look at the top-right corner where the profile icon is located.
- Observe that there is no dark mode toggle option available.

### Expected Behavior
- A toggle switch should be present in the UI, allowing users to switch between dark and light modes.

### Screenshots

<img width="1470" alt="Image" src="https://github.com/user-attachments/assets/f68094e6-0860-45f6-8a79-7bf3e0437a84" />

<img width="1469" alt="Image" src="https://github.com/user-attachments/assets/f6be0ac8-12ab-42bc-8138-530fa30a2db9" />

### Environment:
- Host OS:
- Browser:

---
<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/layer5/5-light-small.svg" width="24px" align="left" /><h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/buttons/community.webp" height="22px" align="left" />Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

**Comment by Radha2786** [comment]
assign this to me

**Comment by GARY121github** [comment]
Go ahead @Radha2786

**Comment by tanishq3105** [comment]
working on it

**Comment by sudhanshutech** [comment]
@Radha2786 did you able to figure out the cause?

**Comment by GARY121github** [comment]
Any updates @Radha2786?
Or should I take over the issue!!!
